src/dex/generic-rfq/generic-rfq-e2e.test.ts

```diff
@@ -85,22 +91,32 @@ const dexKey = 'YOUR_NAME';
 describe(`GenericRFQ ${dexKey} E2E`, () => {
   for (const [_network, testCases] of Object.entries(testConfig)) {
     const network = parseInt(_network, 10);
+    const tokens = Tokens[network];
     const smartTokens = SmartTokens[network];
     const config = generateConfig(network);
 
     config.rfqConfigs[dexKey] = buildConfigForGenericRFQ();
     describe(`${Network[network]}`, () => {
       for (const testCase of testCases) {
-        if (!smartTokens.hasOwnProperty(testCase.srcToken)) {
-          throw new Error(`Please add "addBalance" and "addAllowance" functions for ${testCase.srcToken} on ${Network[network]} (in constants-e2e.ts).`);
-        }
-        if (!smartTokens.hasOwnProperty(testCase.destToken)){ 
-          throw new Error(`Please add "addBalance" and "addAllowance" functions for ${testCase.destToken} on ${Network[network]} (in constants-e2e.ts).`);
-        }
-        const srcToken = smartTokens[testCase.srcToken];
-        const destToken = smartTokens[testCase.destToken];
+        let srcToken: Token | SmartToken, destToken: Token | SmartToken;
+
+        if (SKIP_TENDERLY) {
+          srcToken = tokens[testCase.srcToken];
+          destToken = tokens[testCase.destToken];
+        } else {
+          if (!smartTokens.hasOwnProperty(testCase.srcToken)) {
```

Comment:
@skypper Maybe it would be better 

if( !smartTokens.hasOwnProperty(testCase.srcToken) || !smartTokens.hasOwnProperty(testCase.destToken)) {
...
}

Reply:
Those are two separate conditional branches, though they look similar.